packages/slice-machine/components/FormFields/Input.js

```diff
@@ -9,50 +10,53 @@ const FormFieldInput = ({
   fieldName,
   fields,
   initialValues,
-}) => (
-  <Box sx={sx}>
-    <Label
-      variant="label.primary"
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      htmlFor={fieldName}
-    >
-      {
+}) => {
+  return (
+    <Box sx={sx}>
```

Comment:
what happens with your style if here we pass extra style ? (i'm guessing the property must be used if it exists 😄 )

Reply:
Just had a look and this property isn't actually used anywhere 😆

Reply:
The sx only affects the box, not the field, so it shouldn't be affected

Reply:
i've just tested it and the fields remain the same